lib/rucio/core/transfer.py

```diff
@@ -961,6 +965,20 @@ def get_transfer_paths(total_workers=0, worker_number=0, partition_hash_var=None
         except InvalidRSEExpression:
             pass
 
+    restricted_read_rses = []
+    try:
+        restricted_read_rses = {rse['id'] for rse in parse_expression('restricted_read=true')}
```

Comment:
Please add to http://rucio.cern.ch/documentation/configuration_parameters
RSE Attributes part